consensus/round/round.go

```diff
@@ -6,60 +6,115 @@ import (
 	"time"
 
 	"github.com/multiversx/mx-chain-core-go/core/check"
+	logger "github.com/multiversx/mx-chain-logger-go"
 
+	"github.com/multiversx/mx-chain-go/common"
 	"github.com/multiversx/mx-chain-go/consensus"
+	"github.com/multiversx/mx-chain-go/errors"
 	"github.com/multiversx/mx-chain-go/ntp"
 )
 
+var log = logger.GetOrCreate("consensus/round")
+
 var _ consensus.RoundHandler = (*round)(nil)
 
+// ArgsRound defines the arguments needed to create a new round handler component
+type ArgsRound struct {
+	GenesisTimeStamp          time.Time
+	SupernovaGenesisTimeStamp time.Time
+	CurrentTimeStamp          time.Time
+	RoundTimeDuration         time.Duration
+	SupernovaTimeDuration     time.Duration
+	SyncTimer                 ntp.SyncTimer
+	StartRound                int64
+	SupernovaStartRound       int64
```

Comment:
i've put `Supernova` in field names, did not find a better name for now, we can change this